pkg/controller/operands/quickStart.go

```diff
@@ -129,57 +129,78 @@ func checkCrdExists(ctx context.Context, Client client.Client, logger log.Logger
 
 func getQuickStartHandlers(logger log.Logger, Client client.Client, Scheme *runtime.Scheme, hc *hcov1beta1.HyperConverged) ([]Operand, error) {
 	crdExists, err := checkCrdExists(context.TODO(), Client, logger)
```

Comment:
We should handle the error here.

Reply:
We do. it returned.

checkCrdExists returns false if error or if no CRD found. If the method returns true, error is nil.

Reply:
Yes. Actually we return now.  This is not a big deal and we can keep it as it is. 

let me share my opinion:

my understanding from the Go world, the returned error has a kind of priority. It says whether something went wrong or not. If it is nil, then you can check the other returned values. Now, we have an assumption about the implementation of checkCrdExists function. We know it won't return  (true, err) if the error is not nil. However, it may change in the future and we cannot blame the person who changes it in this way. He/She can say "Hey. I am returning an error. Why don't you respect it.".  Therefore, this implementation puts another responsibility into shoulders of the maintainers of checkCrdExists function, which doesn't seem so good to me.